content/Kotlin/kotlin.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'Kotlin'
+Description: 'Kotlin is a contemporary language released by JetBrains in 2011. It is becoming popular and quickly replacing Java as the choice for Android software projects.'
```

Comment:
Ideally, the `Description` should be one sentence that describes what Kotlin is and is commonly used for.

```suggestion
Description: 'Kotlin is a programming language used for general purposes including front-end, back-end, and mobile development.'
```